test/test_custom_yaml_loader.yaml

```diff
@@ -0,0 +1,65 @@
+bool:
```

Comment:
I know you kinda based that off the preexisting project design, but could I ask you to move all the test data (`.yml` and `.json` files) to a separate folder within tests (like `fixtures`, `data` or something to that extent)? It would be much appreciated.

Reply:
Moved all files to data folder